src/Integration.Vsix/InfoBar/InfoBarManager.cs

```diff
@@ -214,39 +248,50 @@ private class PrivateInfoBarWrapper : IInfoBar, IVsInfoBarUIEvents
             private uint? cookie;
             private bool isClosed;
 
-            public PrivateInfoBarWrapper(IVsWindowFrame frame, IVsInfoBarUIElement uiElement)
+            public PrivateInfoBarWrapper(IVsInfoBarHost host, IVsInfoBarUIElement uiElement)
```

Comment:
Changed the wrapper so it works more generically with the `host`, so it can handle both the ToolWindow and main window cases.